python/lsst/daf/butler/registries/sqliteRegistry.py

```diff
@@ -65,6 +65,7 @@ def setConfigRoot(cls, root, config, full):
             from defaults when Butler instances are constructed
             should be copied from `full` to `Config`.
         """
+        super(SqliteRegistry, cls).setConfigRoot(root, config, full)
```

Comment:
If `super().setConfigRoot()` is not going to work it might be worth having a comment here explaining why you are using the explicit form.

Reply:
Ah, I thought `super()` wouldn't work in a `classmethod`, but it seemed I was wrong about that.